packages/permissions/src/components/authorized/authorized.spec.tsx

```diff
@@ -212,6 +212,7 @@ describe('rendering', () => {
       describe('when general permissions is manage', () => {
         beforeEach(() => {
           props = createTestProps({
+            demandedPermissions: ['ManageOrders'],
```

Comment:
this test broke after applying the change.

This worked previously because the demandedPermission was `ViewProduct` and since there was no match, we did not do any early return and where we "fall back" to checking through `allAppliedDataFences` (which has canManageOrders`